amethyst_assets/src/progress.rs

```diff
@@ -187,14 +199,18 @@ impl Tracker for () {
         asset_name: String,
         error: Error,
     ) {
-        error!(
-            "error loading handle {}, {}, with name {}, caused by: {:?}",
-            handle_id, asset_type_name, asset_name, error
+        let mut err_out = format!(
```

Comment:
Since this is the same piece of code repeated twice in the same file, wouldn't it be better to put it inside of a separate function?

Reply:
very true

Reply:
we saved 7 lines yay. 
I thought we would actually gain code from this.